epub34/authoring/index.html

```diff
@@ -221,12 +221,23 @@ <h4>Relationship to HTML</h4>
 						the various changes to HTML and the technologies it references to ensure they keep their
 						processes up to date.</p>
 
+					<p>The [[html]] standard also combines what used to be two separate technologies: [[?html401]] and
+						[[?xhtml11]]. These two standards differed not only in their syntaxes, but also in what elements
+						and attributes authors could use, as well as how the syntaxes were identified (their
+							<code>DOCTYPE</code> declarations).</p>
+
+					<p>The [[html]] standard now defines a single content model with rules for expressing this tag set
+						using either the <a data-cite="html#syntax">HTML syntax</a> (what has always been considered
```

Comment:
```suggestion
					<p>The [[html]] standard defines a single content model with rules for expressing a tag set
						using either the <a data-cite="html#syntax">HTML syntax</a> (what has always been considered
```

(This is relevant only if the previous paragraph is removed.)